### ISSUE TITLE
cocoapods support

### ISSUE THREAD

**Issue by andrew-skrypnyk** [leantegra/iOSWiBeatSDK#3]
Do you have in plan add cocoapods and carthage support?

**Comment by leantegra** [comment]
yes, we have plan to add cocoapods in our upcoming release in ~1-1.5 month.

**Comment by Petro-Lomaka** [comment]
It's May 18th of 2018. Still no cocoapods support.

**Comment by Petro-Lomaka** [comment]
@leantegra Do you plan to add cocoapods support this year ?

**Comment by yuryks** [comment]
@leantegra any progress on adding cocoapods support ?
